dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.cxf.common-3.2.feature

```diff
@@ -6,12 +6,15 @@ singleton=true
 IBM-App-ForceRestart: uninstall, \
  install
 Subsystem-Name: Internal Apache CXF 3.2 Common Feature for JAX-RS and JAX-WS
--bundles=com.ibm.ws.org.apache.xml.resolver.1.2, \
- com.ibm.ws.org.apache.neethi.3.0.2, \
+-bundles=\
+ com.ibm.websphere.org.osgi.service.http; location:="dev/api/spec/,lib/", \
+ com.ibm.ws.cxf.client, \
  com.ibm.ws.org.apache.cxf.cxf.core.3.2, \
  com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2, \
+ com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.hc.3.2, \
+ com.ibm.ws.org.apache.neethi.3.0.2, \
  com.ibm.ws.org.apache.ws.xmlschema.core.2.0.3, \
- com.ibm.websphere.org.osgi.service.http; location:="dev/api/spec/,lib/"
+ com.ibm.ws.org.apache.xml.resolver.1.2
```

Comment:
Dropped the `com.ibm.ws.cxf.client` bundle down into the common-3.2 feature that we share with jaxrs-2.1, and alphabetized the order.

Reply:
I've also moved `com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.hc.3.2` here as well.